src/core/operations_problem.jl

```diff
@@ -579,17 +564,13 @@ function write_data(psi_container::PSIContainer, save_path::AbstractString; kwar
     return
 end
 
-function write_data(op_problem::OperationsProblem, save_path::AbstractString; kwargs...)
+function write_data(op_problem::OperationsProblem, save_path::String; kwargs...)
```

Comment:
@daniel-thom any reason to prefer AbstractString to String here? Should we change it or revert to AbstractString